src/HashBus.Twitter.Monitor.Simulator/App.cs

```diff
@@ -1,23 +1,41 @@
 namespace HashBus.Twitter.Monitor.Simulator
 {
+    using System;
+    using System.Threading.Tasks;
+    using HashBus.Application.Commands;
     using HashBus.NServiceBusConfiguration;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static void Run(string nserviceBusConnectionString, string hashtag, string endpointName)
+        public static async Task Run(string nserviceBusConnectionString, string hashtag, string endpointName)

```

Comment:
This method no longer calls `Simulator.Start`, so the simulation will never run.